### PR TITLE
{vis}[intel/2016b] mympingpong v0.7.0 w/ Python 2.7.11 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

requires ~~#2165~~, ~~#2470~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 2 out of 4 (4 easyconfigs in this PR)
Linux centos linux 7.1.1503, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/7d546cb56d0d1087155c for a full test report.

**Comment by boegel** [comment]
Jenkins: test this please

**Comment by boegel** [comment]
@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/e7214b248f451cb408f6 for a full test report.

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2016a-Python-2.7.11-timed-pingpong.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+homepage = 'https://bitbucket.org/mpi4py/mpi4py'
+description = """MPI for Python (mpi4py) provides bindings of the Message Passing Interface (MPI) standard for
+ the Python programming language, allowing any Python program to exploit multiple processors."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = [
+    BITBUCKET_DOWNLOADS,
+    GOOGLECODE_SOURCE,
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2016a-Python-2.7.11-timed-pingpong.eb]: deprecated?

**Comment by wpoely86** [comment]
lgtm

File: easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2016a-Python-2.7.11-timed-pingpong.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/mpi4py/mpi4py-1.3.1-intel-2016a-Python-2.7.11-timed-pingpong.eb]: yeah, I might as well drop that, it's worse than deprecated

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 3 out of 5 (3 easyconfigs in this PR)
Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2b00e6b73581f16fb5c2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/bd3a17fb511fc845aa2c for a full test report.

**Comment by boegel** [comment]
going in, thanks for the review @wpoely86!

**Pull request merged**